library/std/src/sys/pal/zkvm/mod.rs

```diff
@@ -6,6 +6,7 @@
 //! This is all super highly experimental and not actually intended for
 //! wide/production use yet, it's still all in the experimental category. This
 //! will likely change over time.
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
Can this be upgraded to `forbid` (like we're trying to do elsewhere) or are there exceptions?

Reply:
done!